Roadmap.md

```diff
@@ -1,188 +1,235 @@
+## Core
+### Update to Node.js 12
+- Leaders: [Ben Newman](https://github.com/benjamn)
+- Status: In Progress
+- PRs: https://github.com/meteor/meteor/pull/10527
 
-The biggest blockers to this project are
+Since Node.js 12 is scheduled to become the LTS version on October 1st, 2019, Meteor 1.9 will update the Node.js version used by Meteor from 8.16.1 (in Meteor 1.8.2) to 12.10.0 (the most recent current version).
 
-1. deciding whether/how to preserve Meteor release versions, and
-2. changing the API of the package server so that you don't have to download the entire package database locally.
+### Tree Shaking
+- Leaders: [Ben Newman](https://github.com/benjamn)
+- Status: -
+- PRs: -
 
-## Page load performance improvements
+Implement tree shaking / dead code elimination, which involves pruning the dependency tree while scanning imports in the `ImportScanner`. We believe it should be possible to treat values like `Meteor.isProduction` as constants during this process, and eliminate those branches if their conditions are false, so that the `./routes` dependency would not show up at all in the production bundle.
 
-*Ongoing*
+### Service worker build target
+- Leaders: <you?>
+- Status: -
+- PRs: -
 
-Speeding up page load times will require a combination of new tools for asynchronous JavaScript delivery (code splitting), dead code elimination, deferred evaluation of JavaScript modules, and performance profiling (so that developers can identify expensive packages).
+A proper service worker build target. Regular Web Workers can be built from a function.toString() but service-workers require an actual server route.
 
-### Making large dependencies optional
+### Ultra-thin Meteor
+- Leaders: <you?>
+- Status: -
+- PRs: -
 
-Making it possible to remove (or dynamically load) large dependencies like `jquery`, `underscore`, and `minimongo` will have a significant impact on bundle sizes.
+Have a ultra-thin version of Meteor that does not depend in anyway to mongo. Allow completely shutting down DDP and dependencies on DDP.
 
-### More aggressive client caching
+### Page load performance improvements
+- Leaders: <you?>
+- Status: -
+- PRs: -
 
-Using `Cache-Control: immutable` to cache the initial JavaScript bundle will reduce the amortized cost of downloading the initial JavaScript bundle in newer browsers.
+Make sure we are not delivering any dependency that is not used
 
-Dynamic `import(...)` benefits dramatically from storing previously-received module versions in [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API), though it may be possible to use IndexedDB in a faster way.
+### Improve Rebuild performance
+- Leaders: <you?>
+- Status: -
+- PRs: -
 
-### Better dead code elimination
+Explore ideas to improve rebuild time such as split main client bundle into several bundles, split the server bundle into several bundles, store less file content in memory, etc
```

Comment:
Disabling the legacy build (at least in dev mode) could also help here for people that don't care about old browsers.

Reply:
Added.

Reply:
Do you want to help @zodern on this as a leader?